actors/src/test/kotlin/ktx/actors/EventsTest.kt

```diff
@@ -79,6 +81,19 @@ class EventsTest {
     assertTrue(listener in actor.listeners)
   }
 
+  @Test
+  fun `should attach ClickListener to a Tree consuming InputEvent with clicked Node`() {
```

Comment:
Please add a test that verifies the selection content (i.e. simulate actual `Tree` usage). Verifying if the listener is in an actor is good first step, but since this logic is specific to a certain widget, we can do more than that to make sure it works correctly.